### DISCUSSION TITLE
3DS《路易吉洋馆》汉化笔记

### DISCUSSION THREAD

**Discussion by giscus-xzonn** [Xzonn/xzonn.github.io#14] [General]
# 3DS《路易吉洋馆》汉化笔记

3DS《路易吉洋馆》汉化笔记 - 来自Xzonn的小站

https://xzonn.top/posts/Luigi-Mansion-Chinese-Localization.html

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****姜半闲 评论于 2019-04-04 04:16:33 (UTC+8)*****

博主删掉了资源？

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****Максим Сапожник 评论于 2020-11-18 14:44:58 (UTC+8)*****

hello, im from russia, great work! i try to translate luigi mansion to russian, can you help me to export and import font from gzf file? may be you share your python script for export? thank you

*****Xzonn 回复于 2020-11-18 17:40:36 (UTC+8)*****

For export, maybe you can use Kuriimu.  
For import, you can find it on my GitHub Gist: https://gist.github.com/Xzonn (main.enlarge - EOJ.py)  
The gzf file in LM is too small. I used the gzf file from Ever Oasis, and it works.

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****Максим Сапожник 评论于 2020-11-24 22:39:32 (UTC+8)*****

Thank you so much!!! Can you share the script for export-import text from gmsg file please?

*****Xzonn 回复于 2020-11-26 11:53:32 (UTC+8)*****

Here is the link: https://github.com/Xzonn/LuigiMansion/blob/master/Text.py

*****Philipe Atanasio 回复于 2022-02-22 03:55:26 (UTC+8)*****

wich comands i need to use, to make it work? export and import text changes

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****romanbbb 评论于 2021-08-29 18:13:26 (UTC+8)*****

Good day, I unpacked ever oasis, took main. gzf (the font is much more extensive), wrote a couple of words in Russian in the main.gmsg file. But the game does not see the great and mighty ((((I suspect that the game needs to be taught to take Russian letters... somehow.... how to do it? maybe who knows ?

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****Philipe Atanasio 评论于 2022-02-22 04:59:53 (UTC+8)*****

hey how can i export .md from main.gmsg ?

**Comment by njsadvance** [comment]
优秀啊，学霸真的干什么都优秀。今天浏览你的小站看到这篇文章有更新，就点进去看了一下。我是环境工程专业，目前从事市政给排水工程设计工作，现在这个行业大环境不好，对未来职业发展有些焦虑，最近刚开始自学python，想着从事个副业或者兼职，所以看了一下你文章里的代码，其实还看不太懂，但是意识到现在这个ai快速发展的时代，不懂点代码就像个文盲。虽然我年纪比你大，但是有很多需要跟你们学习的地方。祝你学业和以后的事业顺利

**Comment by RRay2004** [comment]
你好，我想將你的漢化轉繁體自用，請問順序是否如下：
1.\texts\zh_Hans\main.md 內容修改為繁體
2.main.md 轉成 main.gmsg (不太清楚如何轉)
3.生成繁體的location.gzf 和 main.gzf (一樣不清楚如何做)

求指教

**Comment by THBD2** [comment]
hi i'm glad the font problem is solved, i'm currently working on my majora mask 3ds translation but i don't know how to edit the font, can you help me use your py, i've exported the image from kuriimu. Thank you very much

**Reply by Xzonn** [reply]
This repo is open source: <https://github.com/Xzonn/LuigiMansion>

**Reply by THBD2** [reply]
thank you, but i mean i want to edit gzf font of major mask 3ds can you show me how to do that i have looked at your py file but still don't understand can you show me step by step, thank you

**Reply by THBD2** [reply]
> This repo is open source: <https://github.com/Xzonn/LuigiMansion>

Hey, should I export the font image from kuriimu to a whole table or each letter? If I export each letter, is the image size 20x22 pixels suitable for majora mask or just for ever oasis?

**Reply by Xzonn** [reply]
@THBD2 This repo was created 7 years ago, and has not been updated in the past year. I am no longer familiar with the specific details. I suggest you check the source code or inquire with ChatGPT.

**Reply by THBD2** [reply]
@Xzonn i tried to analyze the py file(main.enlarge - EOJ.py) and found it needs main.md and how the png image of the desired characters may need gb2312 or not. but i found that the eo font when opened with kukki will not display correctly but the majora mask font is ok but the thing is if so then how to redraw the eo font while it is so bad and can i apply this py file to the majora mask font.

**Reply by THBD2** [reply]
@Xzonn   something wrongcan you tell me what is the difference between ever oasis font and majora mask font i cant create majora mask font because of wrong structure i was able to create ever oasis font but i think it cant be used with majora mask i need more information. Thanks
